app/AccountancyModule/Components/Participants/ParticipantList.php

```diff
@@ -224,25 +226,36 @@ private function massEditSubmitted(SubmitButton $button): void
 
         $values = $button->getForm()->getValues()['edit'];
 
-        $changes = [];
+        $changes             = [];
+        $currentParticipants = [];
+        foreach ($this->currentParticipants as $key => $p) {
+            $currentParticipants[$p->id] = $p;
+        }
+
         foreach ($button->getForm()->getValues()->participantIds as $participantId) {
+            $participant = $currentParticipants[$participantId] ?? throw new ParticipantNotFound('Cannot find participant from the given data');
+
             if ($values['days'] !== null) {
-                $changes[] = new UpdateParticipant($this->aid, $participantId, UpdateParticipant::FIELD_DAYS, $values['days']);
+                if ($this->isOnlineLogin && ! $participant->isAccepted()) {
+                    $this->flashMessage(sprintf('Nelze upravit hodnotu dny na táboře. Účastník %s není na tábor přihlášen.', $participant->displayName), 'warning');
```

Comment:
Tohle může vygenerovat 30x flash message, to bych raději spojit do jedné, kde budou jen ty osoby vyjmenované, at je to kratsí.

Reply:
upraveno posledním commitem